source/compatibility.txt

```diff
@@ -26,27 +26,27 @@ MongoDB v4.0 or later.
 
 .. note::
 
-   We recommend that you only use the {+mkc+} with an official version
+   We recommend that you only use the {+connector+} with an official version
    of the MongoDB server. We cannot guarantee it functions correctly
    with any other version.
 
 Kafka Compatibility
 -------------------
 
-The {+mkc+} requires Kafka Connect v2.1.0 or later. It's
-tested against Apache Kafka v2.3 and later. You can use the Kafka
-Connect service with several Apache Kafka compatible platforms including
-the following:
+The {+connector+} requires {+kafka-connect+} v2.1.0 or later. It's
+tested against {+kafka+} v2.3 and later. You can use the
+{+kafka-connect+} service with several {+kafka+} compatible platforms
+including the following:
 
 - Confluent Platform v5.3 or later
 - Microsoft Azure Event Hubs
 - Red Hat AMQ Streams
 
-The {+mkc+} works directly with {+kafka-connect+}. The connector does not
+The {+connector+} works directly with {+kafka-connect-long+}. The connector does not
 connect directly to a Kafka cluster which means it's compatible with any
-Apache Kafka platform that supports {+kafka-connect+}.
+{+kafka+} platform that supports {+kafka-connect-long+}.
```

Comment:
Suggestion:
We should use the full product name only in the first mention.
[Style guide link with highlighting!](https://www.mongodb.com/docs/meta/style-guide/style/product-names-version-numbers/#:~:text=Use%20the%20full%20MongoDB%20product%20name%20on%20the%20first%20mention.%20Subsequent%20mentions%20may%20use%20the%20product%20name%20without%20MongoDB.
)